src/POData/ObjectModel/CynicDeserialiser.php

```diff
@@ -115,9 +148,14 @@ protected function processEntryContent(ODataEntry &$content)
             assert(isset($source), get_class($source));
             $result = $this->getWrapper()->updateResource($set, $source, $key, $properties);
         }
-
         assert($key instanceof KeyDescriptor, get_class($key));
         $content->id = $key;
+
```

Comment:
Needed to add this to enable grandchild elements to be processed, regardless of whether intermediate was ODataEntry or ODataFeed